python_modules/dagster-externals/setup.py

```diff
@@ -36,9 +36,11 @@ def get_version() -> str:
         "typing_extensions>=4.4.0",
     ],
     extras_require={
+        "s3": ["boto3", "botocore"],
```

Comment:
if we go this route should probably just be "boto" or "aws" - these same deps could power any other aws/boto powered direct integration